com.github.sdbg.debug.ui/plugin.xml

```diff
@@ -1077,6 +1079,15 @@
             <bundle name="com.github.sdbg.debug.core"/>
       </component>
    </extension>
+   
+   <!-- Used to launch chrome from another plugin like GWT -->
+   <extension
+         point="com.github.sdbg.debug.ui.launchChrome">
+      <extensions
+            class="com.github.sdbg.ui.launchers.ChromeLaunchShortcutExt">
+      </extensions>
+   </extension>
+
```

Comment:
This implemented the extension point so I could call it from the gwt plugin...

![screen shot 2015-05-05 at 9 16 25 am](https://cloud.githubusercontent.com/assets/1326504/7477266/85e67e48-f307-11e4-8b29-6647827bd3e4.png)